docs/spec_draft.md

```diff
@@ -484,6 +485,62 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cholesky
+
+### Semantics
+
+Computes the Cholesky decomposition of a batch of matrices.
+
+More formally, for all `i`, `result[i0, ..., iR-3, :, :]` is a Cholesky
+decomposition of `a[i0, ..., iR-3, :, :]`, in the form of either of a
+lower-triangular (if `lower` is `true`) or upper-triangular (if `lower` is
+`false`) matrix. The output values in the opposite triangle, i.e. the strict
+upper triangle or strict lower triangle correspondingly, are
+implementation-defined.
+
+If there exists `i` where the input matrix is not an Hermitian
+positive-definite matrix, then the result is implementation-defined.
+
+### Inputs
+
+| Name    | Type                                       |
+|---------|--------------------------------------------|
+| `a`     | tensor of floating-point or complex type   |
+| `lower` | 0-dimensional tensor constant of type `i1` |
+
+### Outputs
+
+| Name     | Type                                     |
+|----------|------------------------------------------|
+| `result` | tensor of floating-point or complex type |
+
+### Constraints
+
+  * (C1) `a` and `result` have the same type.
+  * (C2) rank(`a`) >= 2.
+  * (C3) dim(`a`, -2) = dim(`a`, -1).
+  * (C4) `lower` is optional with default value `false`.
```

Comment:
I think we had a PR review in September where we decided to not bring up default values in the spec, but I cannot recall when or where exactly that was.

To recount the rationale from back then, I think that default values are an implementation detail of the StableHLO API, so I don't think the spec should be prescriptive about then, especially since we don't yet know how the API is going to look like (see #30, #31 and #32 which we deprioritized for the time being).

I'm sure that we'll figure this out later, and maybe default values will make a resurgence in the spec, but for now let's not include them.